privacyidea/static/components/audit/views/audit.log.html

```diff
@@ -101,30 +115,32 @@
             </thead>
             <tbody>
             <tr ng-repeat="audit in auditdata.auditdata">
-                <td>{{ audit.number }}</td>
+                <td ng-if="audit_columns.includes('number')">{{ audit.number }}</td>
                 <td>{{ audit.date | date:dateFormat }}</td>
-                <td>{{ audit.action }}</td>
-                <td><span status-class="{{ audit.success }}">
+                <td ng-if="audit_columns.includes('action')">{{ audit.action }}</td>
+                <td ng-if="audit_columns.includes('success')"><span status-class="{{ audit.success }}">
                 {{ audit.success }}
             </span></td>
-                <td>{{ audit.action_detail }}</td>
+                <td ng-if="audit_columns.includes('action_detail')">{{ audit.action_detail }}</td>
                 <td>
                     <a ng-repeat="serial in audit.serial_list" ui-sref="token.details({tokenSerial:serial})"
                        ng-click="$rootScope.returnTo=audit;">
                     {{ serial }}</a>
                 </td>
-                <td>{{ audit.token_type }}</td>
-                <td>{{ audit.administrator }}</td>
-                <td><a ui-sref="user.details({username:audit.user,
-                realmname:audit.realm})">
+                <td ng-if="audit_columns.includes('token_type')">{{ audit.token_type }}</td>
+                <td ng-if="audit_columns.includes('administrator')">{{ audit.administrator }}</td>
+                <td ng-if="audit_columns.includes('user')">
+                    <a ui-sref="user.details({username:audit.user,realmname:audit.realm})">
                     {{ audit.user }}</a></td>
-                <td><a ng-if="loggedInUser.role == 'admin'" ui-sref="config.realms.list">{{ audit.realm }}</a>
+                <td ng-if="audit_columns.includes('realm')">
+                    <a ng-if="loggedInUser.role == 'admin'" ui-sref="config.realms.list">{{ audit.realm }}</a>
                     <span ng-if="loggedInUser.role == 'user'">{{ audit.realm }}</span>
                 </td>
-                <td><a ng-if="loggedInUser.role == 'admin'" ui-sref="config.resolvers.list">{{ audit.resolver }}</a>
+                <td ng-if="audit_columns.includes('resolver')">
+                    <a ng-if="loggedInUser.role == 'admin'" ui-sref="config.resolvers.list">{{ audit.resolver }}</a>
                     <span ng-if="loggedInUser.role == 'user'">{{ audit.resolver }}</span>
                 </td>
-                <td ng-if="loggedInUser.role == 'admin'">
+                <td ng-if="audit_columns.includes('policies')" ng-if="loggedInUser.role == 'admin'">
```

Comment:
You can connect this via
  
    ng-if="audit_columns.included('policies') && loggedInUser.role == 'admin'

It is not clear what happens with two ng-ifs.

Reply:
done, thank's